server/src/main/java/org/apache/druid/catalog/model/table/export/ExportDestination.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.catalog.model.table.export;
+
+import org.apache.druid.catalog.model.table.IngestDestination;
+
+public interface ExportDestination extends IngestDestination
```

Comment:
Do present changes support local disk file export as well?

Reply:
Also, is this level of indirection required? Are we planning on adding some methods here?

Reply:
We support local disk export, however, it is currently only added in the test module. This would only be used for debugging or for tests.

This class has also been edited to instead contain more methods. Initially, it was added so that ExportDestination could be used directly by MSQ instead of adding an instanceOf check everywhere.